Makefile

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+doc:
+	docker run --rm -v $(CURDIR):/data -u $(id -u):$(id -g) -it chengpan/mdbook:0.2.1 mdbook build ./book
```

Comment:
Can you work with @lpabon to get the `chengpan/mdbook:0.2.1` and its source checked in to an official location?

Reply:
Could you elaborate on where is an official location? currently, I host it in my personal [dockerhub](https://hub.docker.com/r/chengpan/mdbook/) and source is [here](https://github.com/leakingtapan/dockerfiles/tree/master/mdbook)